crates/storage/db/src/metrics.rs

```diff
@@ -258,17 +258,23 @@ impl Labels {
 #[derive(Metrics, Clone)]
 #[metrics(scope = "database.transaction")]
 pub(crate) struct TransactionMetrics {
+    /// Total number of opened database transactions (cumulative)
+    opened_total: Counter,
+    /// Total number of closed database transactions (cumulative)
+    closed_total: Counter,
     /// Total number of currently open database transactions
     open_total: Gauge,
```

Comment:
do we then even need this because this just tracks the diff opened_total - closed_total @Rjected

Reply:
yeah we shouldn't need this now, will need to update the dashboard though

Reply:
will you open an issue @Rjected and close this pr and its issue if no longer a problem pls